src/System.Collections.Specialized/src/MatchingRefApiCompatBaseline.txt

```diff
@@ -0,0 +1,3 @@
+Compat issues with assembly System.Collections.Specialized:
+TypesMustExist : Type 'System.Collections.Specialized.ListDictionary.DictionaryNode' does not exist in the implementation but it does exist in the contract.
```

Comment:
@ViktorHofer this seems to be public for serialization purposes can you please confirm?

Reply:
Correct, that's because of serialization.